File: CX/ClientApp/src/servicios/ServicioTipoIdentificacion.js
import { ProcesarDatosApi } from "./ApiFetch";
const controlador = "tipoidentificacion";

export const AgregarTipoIdentificacion = async (data) => {
    const url = `${controlador}/insertartipoidentificacion`;
    return await ProcesarDatosApi('POST', url, data);
}

export const ActualizarTipoIdentificacion = async (data) => {
    const url = `${controlador}/actualizartipoidentificacion`;
    return await ProcesarDatosApi('PUT', url, data);
}

export const InactivarTipoIdentificacion = async (id) => {
    const url = `${controlador}/eliminartipoidentificacion?idTipoIdentificacion=${id}`;
    return await ProcesarDatosApi('DELETE', url);
}

export const ObtenerTipoIdentificacionPorId = async (id) => {
    const url = `${controlador}/obtenertipoidentificacionporid/${id}`;
    return await ProcesarDatosApi('GET', url);
}

export const ObtenerTipoIdentificacion = async () => {
    const url = `${controlador}/obtenertipoidentificacion`;
    return await ProcesarDatosApi('GET', url);
}
export const ObtenerTipoIdentificacionActivos = async () => {
    const url = `${controlador}/obtenertipoidentificacionactivos`;
    return await ProcesarDatosApi('GET', url);
}

File: BussinesLogic/Servicios/ServicioTipoIdentificacion.cs
using DataAccess.StoredProcedures;
using Dominio.Dto;
using Dominio.Entiti;
using Dominio.Interfaces.Aplicacion.TipoIdentificacion;
using Dominio.Interfaces.Infraestructura.BaseDatos;
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading.Tasks;

namespace BussinesLogic.Servicios
{
    public class ServicioTipoIdentificacion : IServicioTipoIdentificacion 
    { 
        private readonly IRepositorioTipoIdentificacion repositorioTipoIdentificacion;

        public ServicioTipoIdentificacion(IRepositorioTipoIdentificacion repositorioTipoIdentificacion)
        {
            this.repositorioTipoIdentificacion = repositorioTipoIdentificacion;
        }
        public async Task<DtoRespuestaSP> InsertarTipoIdentificacion(EntitiTipoIdentificacion entitiTipoIdentificacion)
        {
            return await this.repositorioTipoIdentificacion.InsertarTipoIdentificacion(entitiTipoIdentificacion);
        }
        public async Task<DtoRespuestaSP> ActualizarTipoIdentificacion(EntitiTipoIdentificacion entitiTipoIdentificacion)
        {
            return await this.repositorioTipoIdentificacion.ActualizarTipoIdentificacion(entitiTipoIdentificacion);
        }
        public async Task<DtoRespuestaSP> EliminarTipoIdentificacion(int idTipoIdentificacion)
        {
            return await this.repositorioTipoIdentificacion.EliminarTipoIdentificacion(idTipoIdentificacion);
        }
        public async Task<DtoTipoIdentificacion> ObtenerTipoIdentificacionPorID(int idTipoIdentificacion)
        {
            return await this.repositorioTipoIdentificacion.ObtenerTipoIdentificacionPorID(idTipoIdentificacion);
        }

        public async Task<List<DtoTipoIdentificacion>> ObtenerTipoIdentificacion()
        {
            return await this.repositorioTipoIdentificacion.ObtenerTipoIdentificacion();
        }

        public async Task<List<DtoTipoIdentificacion>> ObtenerTipoIdentificacionActivos()
        {
            return await this.repositorioTipoIdentificacion.ObtenerTipoIdentificacionActivos();
        }
    }
}